cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help

```diff
@@ -6,40 +6,40 @@ SYNOPSIS
     [--repository home]
 
 DESCRIPTION
-     Uploads the given module to Ballerina Central
+     Upload the given module to the Ballerina Central
```

Comment:
Do we need the "the" here?

For me it's like
"Send it to the WSO2."

Reply:
Fixed.